packages/gatsby-plugin-now/gatsby-node.js

```diff
@@ -0,0 +1,33 @@
+const path = require("path");
+const writeFile = require("util").promisify(require("fs").writeFile);
+
+const REDIRECT_FILE_NAME = "__now_routes.json";
```

Comment:
This file sometimes uses `'` and sometimes `"` for strings, I think it would be good to make it all equal